daemons/attrd/attrd_cib.c

```diff
@@ -591,15 +596,11 @@ write_attribute(attribute_t *a, bool ignore_delay)
             continue;
         }
 
-        /* Remember the XML ID and let peers know it (in case one of them
-         * becomes the writer later)
-         */
-        if (!pcmk__str_eq(v->node_xml_id, node_xml_id, pcmk__str_none)) {
+        if (!pcmk__str_eq(prev_xml_id, node_xml_id, pcmk__str_none)) {
             crm_trace("Setting %s[%s] node XML ID to %s (was %s)",
                       a->id, v->nodename, node_xml_id,
-                      pcmk__s(v->node_xml_id, "unknown"));
-            pcmk__str_update(&(v->node_xml_id), node_xml_id);
-            attrd_broadcast_value(a, v);
```

Comment:
Is it okay that we're no longer telling peers what we learned? (It might have to be...)

Reply:
Yes, because it doesn't need to be per-value now. The broadcast was a temporary hack between commits in this PR to highlight a big benefit of the global cache.

Reply:
Right, it's not global to the cluster though, only global to this node. That was my concern.

I haven't traced the info sources and knowledge state of each node's `attrd`, so I take your word for it if you say it's fine.